packages/css/src/global/global.scss

```diff
@@ -1,51 +1,5 @@
 @include box-sizing();
 
```

Comment:
Er staan nog 2 dingen in, die wel nodig zijn. Wat gaan we met deze file doen, @tfrijsewijk ?

Reply:
We hebben deze file in overleg opgedoekt